protocols/gossipsub/src/peer_score.rs

```diff
@@ -24,12 +24,12 @@
 use crate::metrics::{Metrics, Penalty};
 use crate::time_cache::TimeCache;
 use crate::{MessageId, TopicHash};
+use instant::Instant;
 use libp2p_identity::PeerId;
 use log::{debug, trace, warn};
 use std::collections::{hash_map, HashMap, HashSet};
 use std::net::IpAddr;
 use std::time::Duration;
-use wasm_timer::Instant;
```

Comment:
Can we remove the `wasm_timer` dependency from this crate now?

Reply:
This requires additional work which I will check later. We can't replace all the `wasm_timer::Instant` with `instant::Instant` in gossipsub because of the type conflicts in some of the gossipsub modules.

Reply:
> This requires additional work which I will check later. We can't replace all the `wasm_timer::Instant` with `instant::Instant` in gossipsub because of the type conflicts in some of the gossipsub modules.

Okay, is this PR by itself functional then? Unfortunately we don't have an wasm _runtime_ tests yet.